src/app/index.css

```diff
@@ -184,27 +156,3 @@ body {
   margin-right: 3px;
 }
 
-@media print {
-  .headerArea,.footerArea,.toolbar {
```

Comment:
do we want general print formatting?

Reply:
It's actually still there, I just moved it closer to the other styles that are similar